internal/connector/internal/api/dbapi/connector_type.go

```diff
@@ -125,7 +125,7 @@ func (ct *ConnectorType) SetLabels(labels []string) {
 func (ct *ConnectorType) JsonSchemaAsMap() (map[string]interface{}, *errors.ServiceError) {
 	schema, err := ct.JsonSchema.Object()
 	if err != nil {
-		return schema, errors.GeneralError("Failed to convert json schema for connector type %q: %v", ct.ID, err)
+		return schema, errors.GeneralError("failed to convert json schema for connector type %q: %v", ct.ID, err)
```

Comment:
How have this kind of needed changes been detected?

I understand that by checking manually and not by running any linter, right?

Reply:
@valeriiashapoval As Miguel asked, did we have this manually or automatically detected?

Reply:
hi @valeriiashapoval please remember to answer to this. Thanks